pkg/metrics/metrics.jsx

```diff
@@ -1797,11 +1804,16 @@ class MetricsHistory extends React.Component {
                         action={<Button onClick={() => cockpit.logout(true)}>{_("Log out")}</Button>}
             />;
 
-        if (cockpit.manifests && !cockpit.manifests.pcp)
+        // on a single machine, cockpit-pcp depends on pcp; but this may not be the case in the beiboot scenario,
+        // so additionally check if pcp is available on the logged in target machine
+        if ((cockpit.manifests && !cockpit.manifests.pcp) || this.pmlogger_service.exists === false)
             return <EmptyStatePanel
                         icon={ExclamationCircleIcon}
                         title={_("Package cockpit-pcp is missing for metrics history")}
-                        action={this.state.packagekitExists ? <Button onClick={() => this.handleInstall()}>{_("Install cockpit-pcp")}</Button> : null}
+                        action={this.state.isBeibootBridge === true
+                            // See https://github.com/cockpit-project/cockpit/issues/19143
+                            ? <Text>{ _("Installation not supported without installed cockpit package") }</Text>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19184-20230808-133457-a0bb430c-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)